lib/task/migrate/arUpgrader120.class.php

```diff
@@ -244,7 +244,7 @@ public function up($version, $configuration, $options)
         {
           foreach ($cols as $col)
           {
-            $sql = sprintf('ALTER TABLE %s MODIFY %s VARCHAR(1024) NOT NULL;', $tablename, $col);
+            $sql = sprintf('ALTER TABLE `%s` MODIFY %s VARCHAR(1024) NOT NULL;', $tablename, $col);
```

Comment:
Nitpick: I'd add backticks around the column name here too, for consistency

Reply:
Lets hope they don't become reserved too :P